lib/omise/OmiseAccount.php

```diff
@@ -34,6 +34,6 @@ public function reload()
      */
     private static function getUrl($id = '')
     {
-        return OMISE_API_URL.self::ENDPOINT.'/'.$id;
+        return \Omise\ApiRequestor::OMISE_API_URL . self::ENDPOINT . '/' . $id;
```

Comment:
Is this tie to `ApiRequestor` really necessary? Would it be better to add the URL prefix over in ApiRequestor before calling? I guess only not if this method is intended to be used externally, which it doesn't look like it is

Reply:
@jonrandy Yeah, not really necessary to have it here. 
This one is just a temporary solution to make this pull request small and minimise the area of code that this change is touching (just to make it easy to review).

Now I'm experiencing either having `\Omise\ApiRouter` or moving this `getUrl` back to `OmiseApiResource`. But for sure that the thing that is going to happen next is to get rid of this `getUrl` method from all of resources' class.